htsohm/htsohm.py

```diff
@@ -248,7 +248,46 @@ def get_all_parent_ids(run_id, generation):
             .filter(Material.run_id == run_id, Material.generation == generation) \
             .distinct() if e[0] != None]
 
-def calculate_mutation_strength(run_id, generation, parent):
+def calculate_percent_children_in_bin(run_id, generation, bin):
+    """Determine number of children in the same bin as their parent.
+
+    Args:
+        self (class): row in material table.
+
+    Returns:
+        Fraction of children in the same bin as parent (self).
+    """
```

Comment:
comment here is wrong. This method REALLY needs a comment so we can understand what it is doing. I think we are: finding ALL materials in the provided generation whose PARENT is in the provided bin. Which I think makes sense. The code looks good. But needs a description and parameter docs.